scripts/ci/classify-examples-failures.py

```
@@ -0,0 +1,57 @@
+not.
+"""
+
+import sys
+import xml.etree.ElementTree as ET
+
+BUDGET_MARKERS = ("budget_exceeded", "budget_scope")
+
+
+def classify(path: str) -> str:
+    try:
+        root = ET.parse(path).getroot()
```

Ruff (0.16.2)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)